libctru/include/3ds/services/y2r.h

```diff
@@ -1,113 +1,361 @@
- * ```
+ * @endcode
  */
 typedef struct
 {
-	u16 rgb_Y;
-	u16 r_V;
-	u16 g_V;
-	u16 g_U;
-	u16 b_U;
-	u16 r_offset;
-	u16 g_offset;
-	u16 b_offset;
+    u16 rgb_Y;
+    u16 r_V;
+    u16 g_V;
+    u16 g_U;
+    u16 b_U;
+    u16 r_offset;
+    u16 g_offset;
+    u16 b_offset;
 } Y2R_ColorCoefficients;
 
+/**
+ * @brief Preset coefficients based on standards for the YUV<->RGB formula.
+ *
+ * For more details refer to @ref Y2R_ColorCoefficients
+ */
+typedef enum
+{
+    COEFFICIENT_ITU_R_BT_601         = 0x0, ///< Coefficients from the ITU-R BT.601 standard with PC ranges.
+    COEFFICIENT_ITU_R_BT_709         = 0x1, ///< Coefficients from the ITU-R BT.709 standard with PC ranges.
+    COEFFICIENT_ITU_R_BT_601_SCALING = 0x2, ///< Coefficients from the ITU-R BT.601 standard with TV ranges.
+    COEFFICIENT_ITU_R_BT_709_SCALING = 0x3, ///< Coefficients from the ITU-R BT.709 standard with TV ranges.
+} Y2R_StandardCoefficient;
+
+/**
+ * @brief Structure used to configure all parameters at once.
+ *
+ * You can send a batch of configuration parameters using this structure and @ref Y2RU_SetConversionParams.
+ *
+ */
+typedef struct
+{
+    Y2R_InputFormat input_format        : 8; ///< Value passed to @ref Y2RU_SetInputFormat
+    Y2R_OutputFormat output_format      : 8; ///< Value passed to @ref Y2RU_SetOutputFormat
+    Y2R_Rotation rotation               : 8; ///< Value passed to @ref Y2RU_SetRotation
+    Y2R_BlockAlignment block_alignment  : 8; ///< Value passed to @ref Y2RU_SetBlockAlignment
+    u16 input_line_width;                    ///< Value passed to @ref Y2RU_SetInputLineWidth
+    u16 input_lines;                         ///< Value passed to @ref Y2RU_SetInputLines
+    Y2R_StandardCoefficient standard_coefficient : 8; ///< Value passed to @ref Y2RU_SetStandardCoefficient
+    u8 unused;
+    u16 alpha;                               ///< Value passed to @ref Y2RU_SetAlpha
+} Y2R_ConversionParams;
+
+
+/**
+ * @brief Initializes the y2r service
+ *
+ * This will internally get the handle of the service, and on success call Y2RU_DriverInitialize.
+ */
 Result y2rInit();
+
+
+/**
+ * @brief Closes the y2r service
+ *
+ * This will internally call Y2RU_DriverFinalize and close the handle of the service.
+ */
 Result y2rExit();
 
+
+/**
+ * @brief Used to configure the input format
+ * @param format The input format to be used by the y2r engine
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetInputFormat(Y2R_InputFormat format);
+
+
+/**
+ * @brief Used to configure the output format
+ * @param format The output format to be used by the y2r engine
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetOutputFormat(Y2R_OutputFormat format);
+
+/**
+ * @brief Used to configure the rotation of the output
+ * @param rotation The rotation angle to be used by the y2r engine
```

Comment:
Calling it "angle" makes it sound to me like it support arbitrary rotations. I'd try to find something else.